dashboard/config/locales/blocks.es-MX.yml

```diff
@@ -1468,9 +1468,9 @@ es-MX:
         text: "crear un nuevo sprite llamado {NAME} \n con el disfraz {COSTUME} \n
           en {LOCATION}"
       aalab_createNewSpriteAnon:
-        text: crear un nuevo sprite {ANIMATION_NAME} en {LOCATION}
+        text: crear un nuevo sprite {ANIMATION\_NAME} en {LOCATION}
       aalab_createNewSpriteAnonReturn:
-        text: aalab crear un nuevo sprite {ANIMATION_NAME} en {LOCATION}
+        text: aalab crear un nuevo sprite {ANIMATION\_NAME} en {LOCATION}
```

Comment:
Hm for some reason these escapes came back.

Reply:
I believe this will happen whenever we have to run the out step multiple times

Reply:
Yeah that was my theory too